stablehlo/dialect/StablehloOps.td

```diff
@@ -2339,13 +2339,35 @@ def StableHLO_CustomCallOp: StableHLO_Op<"custom_call",
     Encapsulates an implementation-defined operation `call_target_name` that
     takes `inputs` and `called_computations` and produces `results`.
 
+    If XLA runtime is enabled for a backend, then custom calls use the runtime
+    custom call calling convention to call into the external functions. This
+    calling convention defines an ABI for encoding arguments, attributes and
+    results.
+
+    Depending on the API version there are two ways to pass extra bits of static
+    information to the external function:
```

Comment:
A bit of this info is XLA specific and a bit confusing - i.e. what does this mean: "`API_VERSION_TYPED_FFI` custom calls only supported if XLA uses XLA runtime."

From an IR perspective, all we need to mention is:

```
Depending on the API version there are two ways to pass extra bits of static
information to the external function:

  1. Use `API_VERSION_TYPED_FFI` which allows passing a dictionary attribute.
  2. Use a previous API version with a StringAttr to encode backend config.
```